packages/schema/index.ts

```diff
@@ -0,0 +1,47 @@
+export default {
+  'link': {
+    type: 'inline',
+    attributes: [
+      'destination',
+      'title'
+    ]
+  },
+  'paragraph': {
+    type: 'paragraph'
+  },
+  'heading': {
+    type: 'block',
+    attributes: [
+      'level'
+    ]
+  },
+  'item': {
+    type: 'block'
+  },
+  'list': {
+    type: 'block',
+    attributes: [
+      'type',
+      'startsAt'
+    ]
+  },
+  'bold': {
+    type: 'inline'
+  },
+  'italic': {
+    type: 'inline'
+  },
+  'quotation': {
+    type: 'block'
+  },
+  'line-break': {
+    type: 'object'
+  },
+  'image': {
+    type: 'object',
+    attributes: [
+      'destination',
```

Comment:
destination = link?

Reply:
destination = src; Maybe this should be renamed source?

This was stolen from the commonmark AST